components/ui/AppDataTable.vue

```diff
@@ -68,4 +69,9 @@ export default class AppDataTable extends Vue {
   border-top: none;
   border-bottom: 1px solid $border-color;
 }
+
+.bx--data-table li {
+  list-style-type: disc;
+  margin-left: 1.2rem;
```

Comment:
For consistent spacing, please try and use the spacing variables.

I'm afraid there isn't one in https://www.carbondesignsystem.com/guidelines/spacing/overview/ that matches exactly `1.2rem`, but I believe `1.5rem` may be a good alternative:

```suggestion
  margin-left: $spacing-06;
```

Reply:
Good catch! thank you!